src/stores/appStore.ts

```diff
@@ -100,6 +102,7 @@ export const useAppStore = defineStore("app", {
                 this.admin2Geojson[countryId] = await getGeojsonFeatures(countryId, level);
             }
 
+            this.admin0Geojson = (await getGeojsonFeatures(countryId, 0))[0];
```

Comment:
theres only one feature, the actual country so just unpacking the 0th index

Reply:
Oh I see - so `admin0Geojson` will in fact be a different structure from `admin1Geojson` and `admin2Geojson`? 1 and 2 being key-value dicts, but admin0Geojson being a single 'value', the one feature, if I understood correctly?

I feel having different structures is likely to cause slight confusion. A solution could be renaming the property to something that does not imply a strong analogy to `adminNGeojson`, e.g renaming to `admin0GeojsonFeature` (or renaming the dicts to use a plural term `geojsons`). What do you think?

Reply:
Maybe I'm just not used to using languages that enforce you to document types - it would be reasonable to say that a confused reader should just refer to storeTypes.ts to find out the structures of the properties.

Reply:
this is a fair point actually, will change the name to `admin0GeojsonFeature` instead!